product_docs/docs/pgd/3.7/harp/01_release_notes/harp2.3.2_rel_notes.mdx

```diff
@@ -0,0 +1,14 @@
+---
+title: "Version 2.3.2"
+---
+
+This is a patch release of HARP 2 that includes fixes for issues identified
+in previous versions.
+
+| Type | Description |
+| ---- |------------ |
+| Bug fix | Multiple fixes related to network partition in BDR DCS. |
+| Bug fix | Fix intermittent harp-manager hang issue. |
+| Bug fix | In Go driver calls, ensure rows.Err() is checked every time after rows.Next() returns false. |
+| Change | Improve the BDR DCS's List method performance. |
+| Change | Upgrade 3rd party dependencies to fix Github dependabot alerts. |
```

Comment:
Should this be included? It's a purely internal change of no impact to users.

Reply:
It depends. Some of these libs have security fixes though that doesn't mean it directly affect HARP. I added it because last time we made a release only for these alerts and then nothing else was there to mention but this.

Reply:
Removed that line.